apps/cyberstorm-storybook/.storybook/preview.js

```diff
@@ -1,11 +1,17 @@
 import React from "react";
 
-import { LinkingProvider, CyberstormProviders } from "@thunderstore/cyberstorm";
+import {
+  LinkingProvider,
+  CyberstormProviders,
+  Footer,
+  Header,
+} from "@thunderstore/cyberstorm";
 import "@thunderstore/cyberstorm-styles";
 import { DapperProvider } from "@thunderstore/dapper";
 import { DapperFake } from "@thunderstore/dapper-fake";
 import { LinkLibrary } from "../LinkLibrary";
 import { SessionProvider } from "../SessionContext";
+import styles from "../../cyberstorm-nextjs/app/RootLayout.module.css";
```

Comment:
Relative imports between packages/apps should be no-go IMO.

Reply:
It is a no-go. I'm not sure what to do with storybook at this moment. I'll remove this, but storybook will be borked.